### DISCUSSION TITLE
Force Feedback on Gripper

### DISCUSSION THREAD

**Discussion by archiebaxter** [cyberbotics/webots#6031] [General]
I am trying to get force feedback data on a coupled linear motor which is used to grip objects as they are lifted off the ground by a robotic arm. However, my issue is that my value for the force used by the gripper mechanism does not change regardless of the weight of the object it is picking up. Does anyone know why this is ? For example, in the version i have attached, the value for the gripper motor force is the same whether the red box is 10, 20, 50 or 100kg. Why is this ??
- btw the world is called 'two finger gripper new' and the controller I'm using is 'two_finger_gripper_controller'
[Two_Finger_Gripper.zip](https://github.com/cyberbotics/webots/files/11112558/Two_Finger_Gripper.zip)

**Comment by archiebaxter** [comment]
@ad-daniel could you help with this please?

**Comment by omichel** [comment]
Duplicate with #6018.
